MLP_Model/MLP_utils/exceptions.py

```diff
@@ -0,0 +1,26 @@
+# define python user-defined exceptions
+
+
+class TrainingValidationTestingSplitError(Exception):
+    """Exception occurs in the case that the training, validation, and testing splits to not add up to 1.0"""
+
+    pass
+
+
+class yDataTypeError(Exception):
```

Comment:
```suggestion
class YDataTypeError(Exception):
```
I noticed that this is the only exception that does not appear as orange. Not sure if this is a problem. It seems like the typical convention for classes is to be capitalized, so consider capitalizing the `Y` in this class.